Dependency/DependencyLoader.php

```diff
@@ -93,13 +93,13 @@ public function importMap (string $basePath, array $dependencyMap) : void
 
         foreach ($dependencyMap as $file => $dependencies)
         {
-            $this->dependencyMap["{$basePath}/{$file}.js"] = \array_map(
-                function (string $file) use ($basePath)
+            foreach ($dependencies as $dependency)
+            {
+                if (\preg_match('~(.*)\.js$~', $dependency))
```

Comment:
Please use `if ("js" === pathinfo($file, PATHINFO_EXTENSION))`

Reply:
Good idea. I've updated the PR, although it's `$dependeny` instead of `$file` in this case :)